mlxtend/feature_selection/sequential_feature_selector.py

```diff
@@ -14,25 +14,73 @@
 import sys
 from copy import deepcopy
 from itertools import combinations
-from sklearn.metrics import get_scorer
+from sklearn.metrics import get_scorer, roc_auc_score, accuracy_score
 from sklearn.base import clone
 from sklearn.base import MetaEstimatorMixin
 from ..externals.name_estimators import _name_estimators
 from ..utils.base_compostion import _BaseXComposition
-from sklearn.model_selection import cross_val_score
+from sklearn.model_selection import StratifiedKFold, cross_val_score
 from joblib import Parallel, delayed
 
 
-def _calc_score(selector, X, y, indices, groups=None, **fit_params):
+def fit_and_score(model, X, y, weights,
+                  train, test,
+                  metric=roc_auc_score):
+    # Fit and predict
+    model_clone = clone(model)
+    model_clone.fit(X[train], y[train], sample_weight=weights[train])
+    y_pred = model_clone.predict(X[test])
+
+    # Score
+    score = metric(y[test], y_pred, sample_weight=weights[test])
+    return score
+
+
+def parallel_cross_val_scores_weighted(model, X, y, weights,
```

Comment:
I don't think the 'parallel' in the function name is necessary. Everything is parallel these days 😬